ql/src/java/org/apache/hadoop/hive/ql/optimizer/calcite/rules/views/HiveAugmentSnapshotMaterializationRule.java

```diff
@@ -139,14 +148,11 @@ public void onMatch(RelOptRuleCall call) {
 
     final RelBuilder relBuilder = call.builder();
     relBuilder.push(tableScan);
-    List<RexNode> conds = new ArrayList<>();
-    final RexNode literalHighWatermark = rexBuilder.makeLiteral(
-        mvMetaTableSnapshot.getSnapshotId(), snapshotIdType(relBuilder), false);
-    conds.add(
-        rexBuilder.makeCall(
-            SqlStdOperatorTable.LESS_THAN_OR_EQUAL,
-            ImmutableList.of(snapshotIdInputRef, literalHighWatermark)));
-    relBuilder.filter(conds);
+    final RexNode literalSnapshotId = rexBuilder.makeLiteral(
```

Comment:
since you'll have to re-trigger build again, should we rename to `snapshotIdLiteral`

Reply:
renamed